backend/src/apiserver/webhook/pipelineversion_webhook.go

```diff
@@ -44,6 +44,7 @@ func init() {
 type PipelineVersionsWebhook struct {
 	Client        ctrlclient.Client
 	ClientNoCache ctrlclient.Client
+	cacheDisabled bool
```

Comment:
As mentioned in a previous comment, we can just remove this from the webhook and the call to `NewV2SpecTemplate` can just use a static value of `false` since the resulting template is not used.

Reply:
I haven't addressed all comments yet (I'll leave the PR as draft in the meantime).
Addressed this one now.